packages/blocks/github-pr-overview/src/app.tsx

```diff
@@ -1,241 +1,296 @@
 import * as React from "react";
 
-import { BlockComponent } from "blockprotocol/react";
-import CircularProgress from "@mui/material/CircularProgress";
+import { BlockComponent, useGraphBlockService } from "@blockprotocol/graph";
+import { Box, CssBaseline, Theme, ThemeProvider } from "@mui/material";
 import { theme } from "@hashintel/hash-design-system";
-import { CssBaseline, ThemeProvider } from "@mui/material";
 import {
-  GithubIssueEvent,
-  GithubPullRequest,
   PullRequestIdentifier,
-  GithubReview,
-  isDefined,
-  GITHUB_ENTITY_TYPES,
-  getGithubEntityTypes,
+  BlockState,
+  LocalState,
+  Actions,
 } from "./types";
 import { GithubPrOverview } from "./overview";
-import {
-  collectPrEventsAndSetState,
-  collectPrsAndSetState,
-  collectReviewsAndSetState,
-  getPrs,
-} from "./entity-aggregations";
-import { PullRequestSelector } from "./pull-request-selector";
 
-export enum BlockState {
-  Loading,
-  Error,
-  Selector,
-  Overview,
-}
+import { getPRDetails, getEntityTypeIdsAndPRs } from "./entity-aggregations";
```

Comment:
Capitalization seems inconsistent, do we have a standard in HASH for this yet? I really am not a fan of capitalized acronyms in PascalCase, especially when you do things like try to convert it to another case (which we do with some data in this block..)

[Rust for example](https://rust-lang.github.io/api-guidelines/naming.html) says 
> In UpperCamelCase, acronyms and contractions of compound words count as one word: use Uuid rather than UUID, Usize rather than USize or Stdin rather than StdIn. In snake_case, acronyms and contractions are lower-cased: is_xid_start.

Reply:
updated the casing here => https://github.com/hashintel/hash/pull/705/commits/3d98977e4140b53471f95443e4edf4f8e6172868